helper_functions.js

```diff
@@ -72,4 +72,57 @@ module.exports = {
     );
     return difference <= 10;
   },
-};
+  /**
+   * The method allows to get a well formatted result for the new course request route.
+   * Each course request record now has a property dateOptions that gathers all the date options in a list. 
+   * @param {*} result This is the list of all the new course requests.
+   * @returns The arranged list of all the new course requests having a property date options.
+   */
+  arrangeResult: function (result) {
```

Comment:
Function should be changed to be more specific. It in fact can even be moved to the course_requests.js file as it really can't and shouldn't be used in any other process.